app/models/custom_setting.rb

```diff
@@ -12,5 +12,7 @@
 #
 
 # RailsSettings Model
-class CustomSetting < RailsSettings::CachedSettings
+class CustomSetting < RailsSettings::Base
+  source Rails.root.join("config/app.yml")
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->